### PR TITLE
DDF-2079 upgrade to karaf 4.0.5

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
upgrades karaf from 4.0.4 to 4.0.5 to get some bug fixes particularly PAXWEB-824
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@coyotesqrl @bcwaters @tbatie @jckilmer @adimka
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@shaundmorris 
@stustison
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by stustison** [comment]
Looks like bamboo just hit the oom issue, but succeeded. 👍

**Comment by stustison** [comment]
We also need a backport of this to 2.9.x

File: platform/admin/core/admin-core-api/pom.xml [hunk 1]
```diff
@@ -73,7 +73,6 @@
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.utils</artifactId>
-            <version>1.2.0</version>
```
Comment (jckilmer) [inline platform/admin/core/admin-core-api/pom.xml]: I don't see a version defined anywhere in the hierarchy, should it be? If I run a `mvn help:effective-pom` I get version 1.8.2, is that the desired version?

File: platform/platform-configuration/pom.xml [hunk 2]
```diff
@@ -56,7 +56,6 @@
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.utils</artifactId>
-            <version>1.2.0</version>
```
Comment (jckilmer) [inline platform/platform-configuration/pom.xml]: Same comment as admin-core-api/pom.xml.

**Comment by bcwaters** [comment]
Successfully hero'd this PR
-Built and installed ddf
-Ran ddf after boot with no problems
-shutdown and restarted ddf to ensure no problems occur

**Pull request merged**